manifests/handler.pp

```diff
@@ -62,6 +62,10 @@
 #   Hash.  Handler subdue configuration
 #   Default: undef
 #
+# [*timeout*]
+#   Hash.  Handler timeout configuration
+#   Default: 10
```

Comment:
You're right, needs to be undef. We've taken the value from the sensu documentation, but the default only applies for tcp handlers (https://github.com/sensu/sensu/blob/master/lib/sensu/server/handle.rb#L63)

Will switch to `undef` here.